.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -111,10 +111,10 @@ Prerequisites:
 - [ ] Test 2
 
 ### Test Coverage
-<!-- Please add the test coverages for all changed files here. You can see this when executing the tests locally (see build.gradle and package.json) or when looking into the corresponding Bamboo build plan. -->
-<!-- The line coverage must be above 90% for changes files and you must use extensive and useful assertions for server tests and expect statements for client tests. -->
-<!-- Note: Use the table below and confirm in the last column that you have implemented extensive assertions for server tests and expect statements for client tests. -->
-<!--       You can use `supporting_script/generate_code_cov_table/generate_code_cov_table.py` to automatically generate one from the corresponding Bamboo build plan artefacts. -->
+<!-- Please add the test coverages for all changed files modified in this PR here. You can use `supporting_script/generate_code_cov_table/generate_code_cov_table.py` to automatically generate the coverage table from the corresponding artefacts of your branch. -->
```

Comment:
Good change, this makes things clearer. I would also add a small section explaining the concrete setup of the Python script. Also, I would raise awareness of flaky tests and add to the template that IntelliJ can use to generate coverage. (Run XY with coverage )

Something along the lines of:
1. Generate a GitHub Token: Go to https://github.com/settings/tokens and generate a token with public repository access
2. Run the script in the Artemis working directory and install possible missing packages using `pip install MISSING_PACKAGE`
3. Insert your token from step 1
4. The code coverage will be copied to your clipboard

Reply:
I agree that adding the link is useful, I have just done that in the respective ReadMe. The PR description is in my opinion not the right place to add such a close description - also as far as I am aware these steps are already described in the ReadMe.

Since it seems that you might have missed the ReadMe, I suggest we now also point out that the read me shall be considered for the setup details.